scripts/terrain/terrain_transform.py

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env python3
+
+# Copyright 2018-2018 the openage authors. See copying.md for legal info.
+
+#
+# Pillow is required for image manipulation. Install with pip:
+#
+#     $ pip install pillow
+#
+# This script transforms an image from cartesian to dimetric projection. An
+# inverse operation is supported. Files for the Genie Engine can be created
+# by using a legacy mode that outputs BMP files.
+
+import math, argparse
+from PIL import Image
+
+parser = argparse.ArgumentParser(description='Transforms an image from cartesian to dimetric projection.')
+parser.add_argument('inputfile')
+parser.add_argument('-i', '--inverse', default=False, action='store_true', help='Transforms from dimetric to cartesian')
+parser.add_argument('-l', '--legacy-mode', default=False, action='store_true', dest='legacy_mode', help='Uses BMP instead of PNG as output format and the color PINK (255,0,255) for background instead of the ALPHA channel')
+args = parser.parse_args()
+
+inputfile = args.inputfile
+inverse = args.inverse
+legacy_mode = args.legacy_mode
+
+org_img = Image.open(inputfile)
```

Comment:
all those lines should be put in a function, either in the `main` or in a `args` function that is called from the `main`.

if you run `pylint` over the file, it will be of further assistance.